packages/conventional-changelog-writer/index.js

```diff
@@ -30,7 +30,11 @@ function conventionalChangelogWriter (context, options) {
     noteGroupsSort: 'title',
     notesSort: 'text',
     generateOn: function (commit) {
-      return semverValid(commit.version)
+      var semverOptions = {}
```

Comment:
There is another option available for SemVer functions, but loose was the only applicable one. In future work, we could support more than just `loose`